common/changes/@gooddata/sdk-ui-all/mkn-rail-5175_2023-11-22-15-10.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@gooddata/sdk-ui-all",
+      "comment": "Dashboard plugins no longer support version lock of the GoodData.UI dependencies. Since now, all GoodData.UI dependencies will be injected to the plugin in their latest version at runtime. Also improve way to specify compatibility of the plugin.",
+      "type": "none"
+    }
+  ],
+  "packageName": "@gooddata/sdk-ui-all"
+}
```

Comment:
Add new line and make git happy

Reply:
Updated... we should likely do it somehow automatically as part of other git commit hooks we have, as this is auto generated.